docs/boards/basic.md

```diff
@@ -0,0 +1,54 @@
+ZJS Basic Board Support
+=======================
+
+* [Introduction](#introduction)
+* [Full Pin Specification](#full-pin-specification)
+* [Sample Apps](#sample-apps)
+
+Introduction
+------------
+Unless a specific board support module has been created for your board, you will
+need to use ZJS "Basic Board Support" to use GPIO, AIO, or PWM features.
+
+Full Pin Specification
+----------------------
+Using basic board support means that rather than referring to pins by the same
+friendly names you find in board documentation (IO3, A2, etc.) you will need to
```

Comment:
this seemed like continuous sentence, "you will need to" should be a new sentence?